### PR TITLE
styles

### PR DESCRIPTION
**Pull request by Olexxandr-S**

### CODE REVIEW

**Comment by OlexxandrS** [comment]
add styles

**Review by Olexxandr-S — COMMENTED** [review]
good

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (filkovsp) [inline .vscode/settings.json]: Hi 
we dont need this file. 
local server pert for VS Code Live server is pretty much individual
could you please either delete this file or add into exclusion with [.gitignore](https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/)

File: index.html [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Web calculator</title>
-    <link href="./styles/keyboard.css" rel="stylesheet">
-</head>
-<body>
-<div id="root">
-
-    </div>
-</div>
+    <link href="style.css" rel="stylesheet" />
```
Comment (filkovsp) [inline index.html]: looks like the whole styles.css is not needed any more. please delete it.
but check if we need styles for body. if yes - we might move them into something like `./styles/common.css`

File: styles/screen.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.screen {
+  width: 388px;
```
Comment (filkovsp) [inline styles/screen.css]: please avoid using exact units for width and height in pixels
rem is better, like you have font-size in keyboard class. coz our UI carries pretty much text information.

**Review by nazary001 — COMMENTED** [review]
go working👍

**Review by filkovsp — APPROVED** [review]
cool

**Pull request merged**